data/mods/Aftershock/maps/city_buildings.json

```diff
@@ -9,6 +9,18 @@
       { "point": [ 0, 0, -1 ], "overmap": "basement_bionic_north" }
     ]
   },
+  {
```

Comment:
wouldn't you need to also adjust regional map settings in the mods you are moving it to?  When I look at the mod sections and see only a single file changed, I went to go look at their regional settings files I see that Aftershock as an example only has regional overlay file.  So iirc you would need to add house_09b to the city overlay in Aftershock and any others.

Reply:
Should be fixed with the last commit. No Hope doesn't need a change there because it already has settings for house9 and 9b